java/java.editor/src/org/netbeans/modules/editor/java/Utilities.java

```diff
@@ -93,6 +94,7 @@
 public final class Utilities {
     
     private static final String ERROR = "<error>"; //NOI18N
+    private static final Pattern LINK_PATTERN = Pattern.compile("<a href='(\\*\\d+)'>(.*?)<\\/a>", Pattern.CASE_INSENSITIVE | Pattern.MULTILINE);
```

Comment:
Nitpick: `(.*?) -> (.*)` (* includes no repetition at all)

Reply:
`.*`  is a greedy quantifier that matches as many characters as possible while `.*?` is a lazy quantifier that matches as few characters as possible.

Reply:
eh, sorry ... forgot about greedy/lazy modifier. thanks.